### DISCUSSION TITLE
v1.6.2-beta3 (temporary)

### DISCUSSION THREAD

**Discussion by sudwebdesign** [InvoicePlane/InvoicePlane#1162] [Future Versions]
Install and login OK with 2 deprecated error of vendor/CI with php-8.2 on https://github.com/InvoicePlane/InvoicePlane/releases/tag/v1.6.2-beta-3

**Comment by sudwebdesign** [comment]
install_tables & upgrade_tables & create_user & complete & sessions/login & ...

```
A PHP Error was encountered

Severity: 8192

Message: Creation of dynamic property CI_URI::$config is deprecated

Filename: core/URI.php

Line Number: 102

Backtrace:

File: /var/www/ip-v1.6.2-beta-3-official/index.php
Line: 329
Function: require_once


A PHP Error was encountered

Severity: 8192

Message: Creation of dynamic property CI_DB_mysqli_driver::$failover is deprecated

Filename: database/DB_driver.php

Line Number: 372

Backtrace:

File: /var/www/ip-v1.6.2-beta-3-official/application/third_party/MX/Loader.php
Line: 186
Function: DB

File: /var/www/ip-v1.6.2-beta-3-official/application/modules/setup/controllers/Setup.php
Line: 229
Function: database

File: /var/www/ip-v1.6.2-beta-3-official/application/modules/setup/controllers/Setup.php
Line: 310
Function: load_ci_database

File: /var/www/ip-v1.6.2-beta-3-official/index.php
Line: 329
Function: require_once
```

**Reply by sudwebdesign** [reply]
Other way (here clients/form), same errors `Creation of dynamic property CI_URI::$config is deprecated` +
```
A PHP Error was encountered

Severity: 8192

Message: Creation of dynamic property CI_DB_mysqli_driver::$failover is deprecated

Filename: database/DB_driver.php

Line Number: 372

Backtrace:

File: /var/www/ip-v1.6.2-beta-3-official/application/third_party/MX/Loader.php
Line: 186
Function: DB

File: /var/www/ip-v1.6.2-beta-3-official/application/core/Base_Controller.php
Line: 55
Function: database

File: /var/www/ip-v1.6.2-beta-3-official/application/core/User_Controller.php
Line: 28
Function: __construct

File: /var/www/ip-v1.6.2-beta-3-official/application/core/Admin_Controller.php
Line: 21
Function: __construct

File: /var/www/ip-v1.6.2-beta-3-official/application/modules/clients/controllers/Clients.php
Line: 28
Function: __construct

File: /var/www/ip-v1.6.2-beta-3-official/index.php
Line: 329
Function: require_once
```

**Comment by sudwebdesign** [comment]
@nielsdrost7 official CI3.1.13 with missing `#[AllowDynamicProperties]` ?

Maybe use #1161 for simplify the future.
If dev run `composer install` get CI3.1.13 and have theses deprecated errors.

**Reply by nielsdrost7** [reply]
> Maybe use https://github.com/InvoicePlane/InvoicePlane/pull/1161 for simplify the future.
If dev run composer install get CI3.1.13 and have theses deprecated errors.

Yep, will do that.
For replacing old unmaintaijed CI I had planned on making version 1.7.0 with of course your #1161 (and maybe some other stuff)

**Comment by nielsdrost7** [comment]
@sudwebdesign thanks for testing, man! If everything is good and I get some green lights, I'll make the final release (finally)

Everything with CI 3.1.13 I wouldn't use with PHP 8.2,I think that's why that fork was created as you've said in your PR #1161 
I just didn't want to replace CI with the maintained fork in the 1.6 versions, that's all
